contrib/hadoop/pom.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (C) 2015 Google Inc.
```

Comment:
You own the copyright too; let's acknowledge that by changing this line into:
`Copyright (C) 2015 The Google Cloud Dataflow Hadoop Library Authors`